gulp_tasks/shell.js

```diff
@@ -4,16 +4,16 @@ const browserSync = require( 'browser-sync' );
 
 module.exports = {
 	yarnInstall() {
-		return gulp.src( '' )
+		return gulp.src( './' )
```

Comment:
Heads up, the gulp-shell package has apparently been "blacklisted" by the gulp developers. And, `return gulp.src( '' )` is a big anti-pattern apparently. I tried a few attempts at using node exec/streams for these tasks instead of shell, but I wasn't successful. For now, this continues to work, but we should probably reevaluate the usage of gulp-shell in the future.